crates/diesel_models/src/business_profile.rs

```diff
@@ -231,6 +231,7 @@ pub struct BusinessProfile {
     pub redirect_to_merchant_with_http_post: bool,
     pub webhook_details: Option<WebhookDetails>,
     pub metadata: Option<pii::SecretSerdeValue>,
+    pub intent_fulfillment_time: Option<i64>,
```

Comment:
if the field already present in v1 don't change the order, lets keep the same field order